.scalafmt.conf

```
@@ -26,3 +26,7 @@ align.tokens."+" = [
     ]
   }
 ]
+
+project.excludeFilters = [
+  "src/main/scala-3/PluginCompat.scala"
+]
```

```suggestion
project.layout = StandardConvention
```

https://scalameta.org/scalafmt/docs/configuration.html#projectlayout

---

nice